app/script/entity/message/Message.coffee

```diff
@@ -226,3 +231,33 @@ class z.entity.Message
     else if @status() isnt updated_status
       return @status updated_status
     return false
+
+  start_ephemeral_timer: =>
+    return if @ephemeral_timer
+    @ephemeral_remaining new Date(@ephemeral_expires()) - Date.now()
```

Comment:
@herrmannplatz What do you think about the new approach? Feel free to 
merge if you think this is ready.